script/format

```diff
@@ -0,0 +1,3 @@
+#/bin/bash
+docker-compose run app mix credo -a
+docker-compose run app mix format
```

Comment:
I wonder if it would nice to alias something shorter like 

```bash
alias "captainslog=docker-compose run app mix"
```

then just do 

```bash
captainslog credo -a
captainslog format
```

Reply:
For some reason, aliases just do not want to work. I think this idea though. This was my attempt and output:
```bash
#/bin/bash
alias captainslog="docker-compose run app mix"
captainslog credo -a
captainslog format
```

```
script/format: line 3: captainslog: command not found
script/format: line 4: captainslog: command not found
```

I will just leave it how it is for now and we can revisit this in the future as more scripts get added.